src/app/component-registry.js

```diff
@@ -17,19 +17,19 @@ import ExpressionHistogramSubscriber from '../components/genes/ExpressionHistogr
 import CellSetExpressionPlotSubscriber from '../components/sets/CellSetExpressionPlotSubscriber';
 
 const registry = {
-  [Component.DESCRIPTION]: DescriptionSubscriber,
-  [Component.STATUS]: StatusSubscriber,
-  [Component.GENES]: GenesSubscriber,
-  [Component.CELL_SETS]: CellSetsManagerSubscriber,
-  [Component.SCATTERPLOT]: EmbeddingScatterplotSubscriber,
-  [Component.GATING]: GatingSubscriber,
-  [Component.SPATIAL]: SpatialSubscriber,
-  [Component.HEATMAP]: HeatmapSubscriber,
-  [Component.LAYER_CONTROLLER]: LayerControllerSubscriber,
-  [Component.CELL_SET_SIZES]: CellSetSizesPlotSubscriber,
-  [Component.GENOMIC_PROFILES]: GenomicProfilesSubscriber,
-  [Component.EXPRESSION_HISTOGRAM]: ExpressionHistogramSubscriber,
-  [Component.CELL_SET_EXPRESSION]: CellSetExpressionPlotSubscriber,
+  [ViewType.DESCRIPTION]: DescriptionSubscriber,
+  [ViewType.STATUS]: StatusSubscriber,
+  [ViewType.FEATURE_LIST]: GenesSubscriber,
```

Comment:
I see that the outward facing description of the names has been upgraded.  I guess this is ok then.  Maybe make an issue then for the rest?